tokio/tests/task_local_set.rs

```diff
@@ -355,6 +369,7 @@ fn with_timeout(timeout: Duration, f: impl FnOnce() + Send + 'static) {
     thread.join().expect("test thread should not panic!")
 }
 
+#[cfg_attr(target_os = "wasi", ignore = "FIXME: Does not seem to work with WASI")]
 #[test]
 fn drop_cancels_remote_tasks() {
     // This test reproduces issue #1885.
```

Comment:
It seems like you have been able to remove the majority of these, but this one is still here.

Reply:
`unwrap()` in `with_timeout()` panics on Wasi